docs/spfx/extensions/get-started/building-simple-field-customizer.md

```diff
@@ -38,10 +38,13 @@ You can follow these steps by watching the video on the SharePoint PnP YouTube C
 
     - **What is your solution name?**: field-extension
     - **Which baseline packages do you want to target for your component(s)?** SharePoint Online only (latest)
+    - **Where do you want to place the files** Use the current folder

```

Comment:
Please remove these questions... as the statement before these bullets says, use the default options for any questions omitted from the list. These three are the default options... so we don't want to list them in this list.

Thanks for the other edit on the wrong extension type.